website/content/commands/index.mdx

```diff
@@ -68,3 +68,5 @@ Operation Options:
       specified to the data source type being used in your configuration. This
       is used for example to set a specific Git ref to run against.
 ```
+
+Running `waypoint -autocomplete-install` will add the waypoint autocomplete capability so you can do `waypoint <tab>` on commands. Running `waypoint -autocomplete-uninstall` will remove it. Please note that this will modify your shell init script, so you will need to reload your shell.
```

Comment:
Nitpick but we like to keep our docs (and most of our files to 80 to 100 characters wide max). Newlines in markdown aren't new paragraphs unless there are two so this keeps the lines together still. :)

Reply:
Got it. Fixed all the other lines too and verified that it looks good on Vercel deployment.